webdriver/tests/support/fixtures_bidi.py

```diff
@@ -558,7 +558,7 @@ async def fetch(
 
         body_arg = ""
         if post_data is not None:
-            body_arg = f"body: {post_data},"
+            body_arg = f"body: JSON.stringify({post_data}),"
```

Comment:
Shouldn't this be the following similar to `headers_arg` above?

```suggestion
            body_arg = f"body: {json.dumps(post_data)},"
```